common-lib/src/main/scala/com/gu/mediaservice/lib/metadata/ImageMetadataConverter.scala

```diff
@@ -126,6 +126,9 @@ object ImageMetadataConverter extends GridLogging {
     DateTimeFormat.forPattern("E MMM dd HH:mm:ss.SSS 'BST' yyyy").withZone(DateTimeZone.forOffsetHours(1)),
     DateTimeFormat.forPattern("E MMM dd HH:mm:ss 'BST' yyyy").withZone(DateTimeZone.forOffsetHours(1)),
 
+    DateTimeFormat.forPattern("yyyyMMdd"),
+    DateTimeFormat.forPattern("yyyyMM"),
+    DateTimeFormat.forPattern("yyyyddMM"),
```

Comment:
```suggestion
```

Reply:
^ as I commented above, I’m not against. But what does this change mean for a “date” that is `20202201`? (pretty sure it errors and errors badly: s3-only, no ES record!)

Reply:
That issue would be better fixed by dropping the field instead of the image.

Reply:
> That issue would be better fixed by dropping the field instead of the image.

Goes without saying! And without me saying that it goes without saying! :-) But! Your change means we will drop them, so the question is: would we rather pretend there is no date or have a date wrong by half a year?